README.md

```diff
@@ -58,6 +58,19 @@ user's home directory.
 
 This feature is in early development and is preview only.
 
+# Logging API calls
+The Level Zero Loader will log all API calls whenever logging level is set to `trace` and
+validation layer is enabled. Following variables need to be set to enable API logging:
+
+`ZEL_ENABLE_LOADER_LOGGING=1`
+
+`ZEL_LOADER_LOGGING_LEVEL=trace`
+
+`ZE_ENABLE_VALIDATION_LAYER=1`
+
+By default logs will be written to the log file, as described above. To print the logs
```

Comment:
Though not incorrect in this review, we may need to update the filename above at line 56 as it appears no longer valid according to current behavior. This may be a good time to update.

Reply:
Are they incorrect? From what I see, we still have LOADER_LOG_FILE_DIRECTORY defined as ".oneapi_logs" and LOADER_LOG_FILE as "ze_loader.log"